assets/src/edit-story/components/keyboardShortcutsMenu/index.js

```diff
@@ -18,79 +18,92 @@
  * External dependencies
  */
 import { useRef, useState, useCallback } from 'react';
-import PropTypes from 'prop-types';
+import styled, { StyleSheetManager } from 'styled-components';
+import stylisRTLPlugin from 'stylis-plugin-rtl';
 import { __ } from '@web-stories-wp/i18n';
 
 /**
  * Internal dependencies
  */
 import { isKeyboardUser } from '../../utils/keyboardOnlyOutline';
-import { useGlobalKeyDownEffect } from '../../../design-system';
-import WithTooltip from '../tooltip';
-import { Placement } from '../popup';
-import Modal from '../modal';
-import { Keyboard as KeyboardShortcutsButton } from '../button';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Icons,
+  useGlobalKeyDownEffect,
+  useFocusOut,
+  Tooltip,
+  TOOLTIP_PLACEMENT,
+} from '../../../design-system';
+import { useConfig } from '../../app';
+import { Popup } from './popup';
 import ShortcutMenu from './shortcutMenu';
 import { TOGGLE_SHORTCUTS_MENU } from './constants';
 
-function KeyboardShortcutsMenu({ onMenuToggled }) {
+const Wrapper = styled.div``;
+
+const withRTLPlugins = [stylisRTLPlugin];
+const withoutRTLPlugins = [];
+
+function KeyboardShortcutsMenu() {
   const anchorRef = useRef();
+  const wrapperRef = useRef();
   const [isOpen, setIsOpen] = useState(false);
+  const { isRTL } = useConfig();
 
-  const toggleMenu = useCallback(
-    (e, showMenu) => {
-      e.preventDefault();
+  const closeMenu = useCallback(() => setIsOpen(false), [setIsOpen]);
 
-      setIsOpen((prevIsOpen) => {
-        const menuOpen = showMenu ?? !prevIsOpen;
+  const toggleMenu = useCallback((e, showMenu) => {
+    e.preventDefault();
+    setIsOpen((prevIsOpen) => {
+      const menuOpen = showMenu ?? !prevIsOpen;
 
-        if (onMenuToggled) {
-          onMenuToggled(menuOpen);
-        }
+      if (isKeyboardUser() && !menuOpen) {
+        // When menu closes, return focus to toggle menu button
+        anchorRef.current.focus?.();
+      }
 
-        if (isKeyboardUser() && !menuOpen) {
-          // When menu closes, return focus to toggle menu button
-          anchorRef.current.focus?.();
-        }
+      return menuOpen;
+    });
+  }, []);
 
-        return menuOpen;
-      });
-    },
-    [onMenuToggled]
-  );
+  const label = __('Open Keyboard Shortcuts', 'web-stories');
 
   useGlobalKeyDownEffect(TOGGLE_SHORTCUTS_MENU, toggleMenu, [toggleMenu]);
+  useFocusOut(wrapperRef, closeMenu, []);
```

Comment:
They should? Oh, didn't see that. Well, that's easy to remove. Fixed in f2aed81fe.

Reply:
The decision on this one was reversed, by the way - to make it consistent with other overlays. And also because the keyboard icon will be moved to the left side, then the popup will conflict with the help center popup and it's easier to just always close it when unfocused.

Reply:
Reverted in ba7ff1282.

Reply:
Ok, good to know.